examples/bme680_simpletest.py

```diff
@@ -10,8 +10,13 @@
 # change this to match the location's pressure (hPa) at sea level
 bme680.sea_level_pressure = 1013.25
 
+# You will usually have to add an offset to account for the temperature of
+# the sensor. This is usually around 5 degrees but varies by use. Use a
+# separate temperature sensor to calibrate this one.
+temperature_offset = -5
+
 while True:
-    print("\nTemperature: %0.1f C" % bme680.temperature)
+    print("\nTemperature: %0.1f C" % bme680.temperature + temperature_offset)
```

Comment:
This will throw. Parentheses should be surrounding the calculation.

`print("\nTemperature: %0.1f C" % (bme680.temperature + temperature_offset))`

Reply:
@houmark Good catch! If you could file an issue, it would be greatly appreciated. Or, if you're comfortable submitting a PR, that would be amazing too!

Reply:
This was actually fixed in a [this commit](https://github.com/adafruit/Adafruit_CircuitPython_BME680/commit/e8689dcc806d5271f6baf06ec5122f35af5f3528), so nothing more to do here ;)